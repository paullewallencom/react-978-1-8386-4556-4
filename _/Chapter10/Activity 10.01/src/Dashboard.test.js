import React from 'react';
import {render} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Dashboard from './Dashboard';
import { MemoryRouter } from 'react-router-dom';
import { createBrowserHistory } from 'history';

test('renders correctly', () => {
  const history = createBrowserHistory();

  history.push("/dashboard");
  const {container} = render(
    <MemoryRouter>
      <Dashboard />
    </MemoryRouter>);

  expect(container).toMatchSnapshot()
});
