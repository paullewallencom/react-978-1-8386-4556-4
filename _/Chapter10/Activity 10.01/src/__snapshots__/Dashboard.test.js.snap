// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="sidebar"
    >
      Sidebar
    </div>
    <div
      class="main"
    >
      <div
        class="navbar"
      >
        <a
          href="//map"
        >
          Map
        </a>
        <a
          href="//chart"
        >
          Chart
        </a>
        <a
          href="//table"
        >
          Table
        </a>
      </div>
      <h3>
        Please select a widget.
      </h3>
    </div>
  </div>
</div>
`;
